profiles/tests/test_migrations.py

```diff
@@ -146,3 +146,29 @@ def verify_migration(apps):
         create_data,
         verify_migration,
     )
+
+
+def test_verified_personal_information_shorter_text_fields_migration(migration_test_db):
```

Comment:
This is all well and good, but do we need to verify Django doing migrations correctly? As there seems to be no custom operations in the migration.

Maybe because they are encrypted fields?

Reply:
I wrote this test because I thought custom migration code would be needed (to shorten any too long values in the database). But when I run this test, I noticed that it didn't fail and Django actually did the shortening automatically. Which was nice but a surprise to me. So no custom code was needed, but I still left the test be, perhaps to ensure that Django keeps on doing what it did.

I wouldn't say we have to verify Django doing migrations correctly, so in that sense this test is needless. I could also remove this test. Perhaps it has served its purpose, which was to educate me about Django's capabilities :smile:

Reply:
Removed that test.